README.md

```diff
@@ -175,33 +150,21 @@ Below are listed tentative CIPs still under discussion with the community. They
 
 ### Proposals Under Review (CPS)
 
-Below are listed tentative CPSs still under discussion with the community. They are listed below for easing navigation and also tentatively allocating numbers to avoid clashes later on.
-
-| #     | Title |
-| ----- | --- |
-| 0001? | [Metadata Discoverability and Trust](https://github.com/cardano-foundation/CIPs/pull/371) |
-| 0004? | [Spending Script Redundant Execution](https://github.com/cardano-foundation/CIPs/pull/418/) |
-| 0006? | [Governance Security](https://github.com/cardano-foundation/CIPs/pull/491) |
-| 0008? | [Domain Name Resolution](https://github.com/cardano-foundation/CIPs/pull/605) |
-| 0012? | [Query Layer Standardization](https://github.com/cardano-foundation/CIPs/pull/625) |
-| 0015? | [Intents for Cardano](https://github.com/cardano-foundation/CIPs/pull/779) |
+The following link lists "candidate" CPSs still under discussion with the community; these are assigned numbers to avoid later clashes and to facilitate community discussion (see further below for stalled proposals):
 
-<p align="right"><i>Last updated on 2024-07-27</i></p>
+**[CPS pull requests under active review](https://github.com/cardano-foundation/CIPs/pulls?q=is%3Apr+is%3Aopen+%2F%5ECPS-%2F+in%3Atitle+label%3A%22State%3A+Confirmed%22%2C%22State%3A+Last+Check%22+draft%3Afalse+-label%3AUpdate%2CCorrection%2CTranslation%2C%22Bi-Weekly+Notes+%2F+Editorial+Housekeeping%22%2C%22CIP-0010%3A+new+registry+entry%22%2C%22CIP-0067%3A+new+label%22%2C%22CIP-0088%3A+new+extension%22)**
 
 ## Stalled / Waiting For Authors
 
-The following list contains proposals that have been under review and for which actions are now awaiting updates from their original authors. Proposals stalled for several months without any updates from their authors and will eventually be closed. However, authors are invited to re-open pull requests or open new ones should they want to bring the discussion back to life.
+The following links list proposals deemed ready for review but requiring further update
```

Comment:
```suggestion
The following links list proposals deemed ready for review but requiring further update.
```
punctuation.

Reply:
@Ryun1 keep reading down to line 160... the sentence & paragraph continue to form the following text (as seen [here in branch](https://github.com/rphair/CIPs/tree/replace-tables-with-state-queries?tab=readme-ov-file#stalled--waiting-for-authors)):
> The following links list proposals deemed ready for review but requiring further update from the original author(s) or other confirmation of proposal elgibility (if considered deprecated):